### DISCUSSION TITLE
mod just stopped working right for me

### DISCUSSION THREAD

**Discussion by Simply-James** [cc-tweaked/CC-Tweaked#1926] [General]
recently ive been having alot of issues with it and one issue is that modems just stopped working. i even copied code from the twaked.cc website and made sure the modem is on the right side but when i try to open it with `rednet.open` it says it couldnt find it and `modem.open` (yes i set the local variable right) it gives a nil value error. even after reinstalling and updating it doesnt work.

**Comment by zyxkad** [comment]
`rednet.open` accepts a string `"right"` or `'right'`, not `right`, and not a peripheral table

**Comment by SquidDev** [comment]
This is most likely caused by Optifine. CC:T *should* show a message when joining a world warning about the presence of Optifine, but it's possible this is not working correctly — could you upload your mod list and logs?
